gradle/suppress.gradle

```diff
@@ -0,0 +1,29 @@
+import groovy.xml.XmlUtil
+
+def suppressCves(reportPath, suppressionsPath) {
+    def report = new XmlSlurper().parse(new FileReader(reportPath))
+    def vulns = report.'**'.findAll { node -> node.name() == 'vulnerability' }.collect { it.name.text() } as Set<String>
+
+    if (vulns) {
+        def file = file(suppressionsPath)
+        def suppressions = new XmlSlurper(false, false).parse(file)
+
+        vulns.each {
```

Comment:
Currently this creates a new `<suppress>` node per vulnerability, as evidenced in the suppression file changes in this PR.

Reply:
Now using DOM elements instead to create each node.
- Provides flexibility to insert different elements.
- Helps format and retain certain xml attributes. Previously comments were stripped after suppression.